docs/.vuepress/tools/snippet-transform/templates/react.js

```diff
@@ -0,0 +1,150 @@
+const beautify = require('js-beautify').js;
+const { indentLines } = require('./helpers');
+
+/**
+ * Return a default set of imports for the React example.
+ *
+ * @param {boolean} includeImports `true` if the final snippet should include the imports section.
+ * @returns {string}
+ */
+function getImportsSection(includeImports) {
+  return includeImports ? `\
+import React, { useEffect } from 'react';
+import ReactDOM from 'react-dom';
+import { HotTable } from '@handsontable/react';
+import { registerAllModules } from 'handsontable/registry';
+
+// register Handsontable's modules
+registerAllModules();
+
+` : '';
```

Comment:
Can we please persistently disable cache then? I guess it is even more problematic if you switch branches and end up reusing cache from an old branch.

Cache speeds up `npm run docs:build` and the initialization of `npm run docs:start`, at the expense of being sometimes incorrect. I think it is unacceptable.

@kirszenbaum do you have an opinion here?

Reply:
Interestingly enough, I don't think I ever used the `:no-cache` option, because I didn't have any problems with cache so far. But if `:no-cache` doesn't slow things down a lot, and is theoretically more accurate, I'd go for it

Reply:
I think the cache kicks in when nothing is modified in the `.md` files, so it probably doesn't matter as much when preparing the actual content.
I'm not sure how much time we save by using the cached script, compared to the non-cached one, though.

Reply:
@warpech @kirszenbaum 
I did some testing, I consecutively ran:
1. `DOCS_VERSION npm run docs:start` - it took around **4 minutes, 54 seconds**
2. `DOCS_VERSION npm run docs:start` - it took around **54 seconds**
3. `DOCS_VERSION npm run docs:start` - it took around **52 seconds**
4. `DOCS_VERSION npm run docs:start` - it took around **52 seconds**
5. `DOCS_VERSION npm run docs:start:no-cache` - it took around **5 minutes 12 seconds**

So my take on this is - depending on the state of the cache, the watch script can speed up up to ~6 times. I think it's a significant difference.

Reply:
OK, let's keep the cache then!